haystack/utils/deepsetcloud.py

```diff
@@ -489,11 +495,36 @@ def deploy(
             headers=headers,
         )
 
+        if workspace is None:
+            workspace = self.workspace
+        if pipeline_config_name is None:
+            pipeline_config_name = self.pipeline_config_name
+
+        pipeline_url = f"{self.client.api_endpoint}/workspaces/{workspace}/pipelines/{pipeline_config_name}/search"
```

Comment:
It might make sense to move this to the (`PipelineClient`) in `haystack.utils` s.t. we have all the urls at one place ?

Reply:
@ArzelaAscoIi  actually this method is part of `PipelineClient`. Or do you mean to introduce another method like `get_pipeline_url()` and which return this?

Reply:
Ouhh! sorry! You are right! NVM